DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: plotly
 Title: Create interactive web-based graphs via plotly's API
-Version: 1.0.9
+Version: 1.1.0
```

Comment:
Yea, i suppose so. In that case, you might want to remove the 1.0 reference on the home page ;) <https://plot.ly/r/>